VRDR/DeathRecord_submissionProperties.cs

```diff
@@ -5416,31 +5419,197 @@ public string DeathLocationTypeHelper
                 {
                     quantity.System = extractedSystem;
                 }
+                if (extractedUnit != null)
+                {
+                    quantity.Unit = extractedUnit;
+                }
                 AgeAtDeathObs.Value = (Quantity)quantity;
             }
         }
 
+        /// <summary>Age At Death Years Helper</summary>
+        /// <value>Set decedent's age at time of death in years.</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleDeathRecord.AgeAtDeathYears = 100;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Age At Death: {ExampleDeathRecord.AgeAtDeathYears} years");</para>
+        /// </example>
+        [Property("Age At Death Years Helper", Property.Types.Int32, "Decedent Demographics", "Age At Death in Years.", false, IGURL.DecedentAge, true, 2)]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='39016-1')", "")]
+        public int AgeAtDeathYears
+        {
+            get {
+                if (AgeAtDeathObs?.Value != null)
+                {
+                    Quantity quantity = (Quantity)AgeAtDeathObs.Value;
+                    if (quantity.Code == "a")
+                    {
+                        return Convert.ToInt32(quantity.Value);
+                    }
+                }
+                return 0; // not sure how to best handle this case
```

Comment:
Does it it make sense to have

```
public int? AgeAtDeathYears
{
    ...
    return null;
}
```

In that case the setter should check for null